faqrenderer/faq.md

```diff
@@ -0,0 +1,121 @@
+
+- pymol and ipymol: Requires extra conda channels and is not free software. Can be installed manually in a new conda environment.
+- opty: Requires Python <=3.6. Can be installed manually in a new conda environment with Python 3.6.
+- rpy2: Requires R <= 4.0. Can be installed manually in a new conda environemt with R 3.6.
+
+If you require the above packages or older versions of existing ones, you should use the legacy environment or [create a conda environment](#how-do-i-create-a-conda-environment) with your custom package versions. Please email us at [jupyterteam@ucdavis.edu](mailto:jupyterteam@ucdavis.edu) if you have concerns or issues with the new environment.
+
+## How do I access RStudio?
+
+To open the RStudio interface, you can select the RStudio notebook icon from the Jupyter Launcher (Ctrl+Shift+L). This will open a new JupyterLab tab which has an RStudio interface and access to all the same packages you would find elsewhere on your account.
+
+![RStudio Launcher icon](RStudio-launcher.png)
+
+## How can I install custom packages?
+
+You are free to install any additional packages available through conda by [creating a conda environment](#how-do-i-create-a-conda-environment), or by running [conda install 'package'](https://docs.conda.io/projects/conda/en/latest/user-guide/concepts/installing-with-conda.html) commands within your custom environment. These packages will persist for you. You may also use `conda install` directly in the `(notebook)` environment provided by default, but these packages won't be permanent and will be wiped anytime your server restarts.
+
+For packages not available through conda or for languages which have their own package manager such as SageMath and Julia, we suggest that you [email us](mailto:jupyterteam@ucdavis.edu) to provide them for you by default in the environment. Packages must be installed to the `/home/jovyan/` directory to persist and Jula and SageMath do not install there by default. For Instructors wanting to use custom packages for their class, [you'll probably want to email us as well](#how-do-i-set-up-custom-environments-for-my-class).
+## How do I create a Conda environment?
+
+The best way to permanently install new packages onto your account is through conda environments. Using conda environments, you may also build Jupyter notebooks and consoles which contain only the programming language and packages that you need. Follow the steps below to create one.
+
+1. In the top left corner, open a terminal by going to File->New->Terminal or open a New Launcher (Ctrl+Shift+L) and select terminal from the 'other' section.
+
+  ![Finding the Terminal](terminal.png)
+
+2. Within the terminal, use the command `conda create -n 'your_env_name' 'kernel' 'packages'` to create your environment.
```

Comment:
```suggestion
2. Within the terminal, use the command `conda create -n <your_env_name> <kernel> <packages>` to create your environment.
```

It think this is a bit more common for noting a placeholder. You can also just point them to the conda documentation on this.